.github/workflows/build.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: microsoft/windows-drivers-rs
-          ref: main
+          ref: refs/pull/597/head
```

Comment:
revert this and other jobs to use main

Reply:
okay. verified all checks pass using the new pr branch. reverting to main (ci will fail until https://github.com/microsoft/windows-drivers-rs/pull/597 merges)